commcare-cloud-bootstrap/bootstrap.sh

```diff
@@ -12,6 +12,24 @@ do :
 done
 
 commcare-cloud $ENV bootstrap-users --quiet --branch=$BRANCH
-commcare-cloud $ENV deploy-stack --skip-check --quiet -e 'CCHQ_IS_FRESH_INSTALL=1' --branch=$BRANCH
 
-commcare-cloud $ENV fab deploy:confirm=no,skip_record=yes --show=debug --set ignore_kafka_checkpoint_warning=true
+commcare-cloud $ENV ansible-playbook update_apt_cache.yml --skip-check --quiet --branch=$BRANCH -e 'CCHQ_IS_FRESH_INSTALL=1'
```

Comment:
can you not extract these args to a var a to avoid copying them on each line? Or else you could use a loop to execute each playbook.